spec/support/inherited_page.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class InheritedPage < ExamplePage
+  inherit_section :foo
+  inherit_section :bar, if: :no_bar
```

Comment:
I think this DSL breaks the Less Surprise Principle. When I inherit a page from another one, I expect the children to have all the parent's sections (sections is the main responsibility for `Tram::Page`) without necessity to explicitly select them.

What I'd like is something like:

```ruby
class InheritedPage < ExamplePage
  # section :foo is already here from `ExamplePage`
  section :bar, if: :no_bar, reload: true # tells explicitly: "I know that :bar is already here, and yes, I want to reload it"
  section :bam
end
```

@gzigzigzeo ?